hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/BackupInfo.java

```diff
@@ -267,7 +267,7 @@ public String getFailedMsg() {
   }
 
   public void setFailedMsg(String failedMsg) {
-    if (failedMsg.length() > MAX_FAILED_MESSAGE_LENGTH) {
+    if (failedMsg != null && failedMsg.length() > MAX_FAILED_MESSAGE_LENGTH) {
```

Comment:
This is an unrelated, but minor, change. I realized that there was a NPE opportunity here.